docs/introduction.md

```diff
@@ -0,0 +1,27 @@
+Introduction
+============
+
+[OpenMS](http://www.openms.org/)
+is an open-source software C++ library for LC-MS data management and
+analyses. It offers an infrastructure for rapid development of mass
+spectrometry related software. OpenMS is free software available under the
+three clause BSD license and runs under Windows, macOS, and Linux.
+
+It comes with a vast variety of pre-built and ready-to-use tools for proteomics
+and metabolomics data analysis (TOPPTools) as well as powerful 1D, 2D and 3D
```

Comment:
Should this be "TOPP tools"

Reply:
Is OpenMS only for users from the proteomics and metabolomics fields or does it have wider applications? I think we should mention this

Reply:
hmm mainly proteomics and metabolomics I would say. In principle, the more general parts can be used for anything using mass spectrometry.

Reply:
Should we say mass spectrometry based data analysis for proteomics, metabolomics, and other omics?